### ISSUE TITLE
spacy_download_langmodel_virtualenv() ignores given envname; overrides with `spacy_virtualenv` 

### ISSUE THREAD

**Issue by olyerickson** [quanteda/spacyr#238]
spacy_download_langmodel_virtualenv() ignores whatever envname one gives it. Simple example:

```
> spacy_download_langmodel_virtualenv(model="en",
+                          envname = "r-spacyr",
+                          virtualenv_root = "/software/anaconda3/envs")

installing model "en"

Error in spacy_download_langmodel_virtualenv(model = "en", envname = "r-spacyr",  : 
  The virtual environemnt /software/anaconda3/envs/spacy_virtualenv does not exist
```

You can see it accepted my virtualenv_root but ignored my envname...

**Comment by olyerickson** [comment]
Also there is a typo in the error message ("environemnt") ;)

**Comment by olyerickson** [comment]
I can actually see in your source code that you ignore `envname` ...

https://github.com/quanteda/spacyr/blob/06b4eadd421a0f94c0645dd4297c0e60eec8545d/R/spacy_langmodel_functions.R

**Comment by olyerickson** [comment]
Furthermore, the implementation of `spacy_download_langmodel_virtualenv()` appears to assume we're using miniconda rather than conda (because it is hard-coding `bin/activate`)

**Comment by kbenoit** [comment]
Should be solved with v1.3.
